modules/gis/test/reproject.spec.js

```diff
@@ -0,0 +1,62 @@
+import test from 'tape-promise/tape';
+import {mapBinaryCoords, mapGeoJsonCoords} from '@loaders.gl/gis';
+import {Proj4Projection} from '@math.gl/proj4';
+
+test('gis#reproject GeoJSON', t => {
+  const projection = new Proj4Projection({from: 'WGS84', to: 'EPSG:3857'});
```

Comment:
If we find a better name, maybe we should rename this class in math.gl?

- `GeospatialReprojection`?
- `CRSProjection`?

etc.